### DISCUSSION TITLE
increase the spawn rate keeping default values

### DISCUSSION THREAD

**Discussion by mikima** [EwyBoy/OreTweaker#78] [Help Wanted!] [answered]
hello, is there a simple way to increase the standard ore spawn (e.g. coal x2, diamonds x10) but keeping all the other values as default (e.g. minX/minY)?

thanks!

**Comment by EwyBoy** [comment, marked as answer]
Ore Tweaker by default generates a file for every vanilla ore with all the vanilla settings. 
So you can easily open each of these files and look at the spawn rate and multiply that value by 2 or 10 or whatever you want. If the only value you change is spawn rate you will end up with vanilla ore settings with the multiplier you set for them. 

But there is no global setting to change the spawn rate for every ore.

**Comment by mikima** [comment]
thanks that's very useful! I thought I had to write each configuration file from scratch
